src/i18n/en-US/action.ts

```diff
@@ -65,9 +65,13 @@ const action = {
       day: 'Day',
       year: 'Year'
     },
+    currentScope: 'Current Scope',
+    currentNextSteps: 'Current Next Steps',
```

Comment:
Are next steps also optional? Noting from the GQL `CreateSystemIntakeActionExtendLifecycleIdInput` input type that they're not required

Reply:
Next steps are not optional, but only the front end enforces it. I believe the original requirements were that it was optional but that got changed although I may be mistaken on that.

Reply:
I can refactor next steps to be not optional in the graphql/backend as well. I think it may be easier to split that out in another story/review since ive seen similiar discepensies in other places